test/Microsoft.IdentityModel.Tokens.Tests/AsymmetricSignatureTests.cs

```diff
@@ -7,12 +7,20 @@
 using Microsoft.IdentityModel.TestUtils;
 using Xunit;
 
-#pragma warning disable CS3016 // Arrays as attribute arguments is not CLS-compliant
 #pragma warning disable SYSLIB0028 // Type or member is obsolete
 #pragma warning disable SYSLIB0027 // Type or member is obsolete
 
 namespace Microsoft.IdentityModel.Tokens.Tests
 {
+    /// <summary>
+    /// This class defines the test collection for all tests that set an AppContext switch.
+    /// All tests in this collection will run sequentially to prevent unexpected behavior
+    /// due to interference between tests as multiple tests set the switch.
+    /// </summary>
+    [CollectionDefinition(nameof(AppContextTestCollection), DisableParallelization = true)]
```

Comment:
Unit tests are failing on PR build and GitHub actions could be related to this. Can consider writing tests on a separate class instead.